libs/ui-watt/src/lib/foundations/icon/watt-icon-state.ts

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
```

Comment:
suggestion: Revert this change so that we avoid touching Watt as part of this pull request.